auto/sendfile

```diff
@@ -156,3 +131,24 @@ if [ $nxt_found = no ]; then
         NXT_HAVE_HPUX_SENDFILE=YES
     fi
 fi
+
+# Linux sendfile().
```

Comment:
This comment similar to other tests in this file should be inside the `if [ $nxt_found = no ]; then` block.